src/clj/rems/css/styles.clj

```diff
@@ -202,6 +204,69 @@
                                   :margin-top 0
                                   :margin-bottom (u/px 16)})
 
+(defn- remove-nil-vals
+  "Recursively removes all keys with nil values from a map."
+  [obj]
+  (assert (not= "" obj))
+  (cond
+    (record? obj) obj
+    (map? obj) (let [m (->> obj
```

Comment:
This could now be simplified further?
```
  (map? obj) (->> obj
                  (map-vals remove-nil-vals)
                  (remove-vals nil?)
                  not-empty)
```